src/Microsoft.VisualStudio.Services.Agent/Util/IOUtil.cs

```diff
@@ -102,6 +102,11 @@ public static string GetProxyConfigFilePath()
             return Path.Combine(GetRootPath(), ".proxy");
         }
 
+        public static string GetAutoLogonSettingsFilePath()
+        {
+            return Path.Combine(GetRootPath(), ".autologonsettings");
```

Comment:
Can you remove "settings" from the file name? Should be consistent with existing pattern. Also, it's arguable that "settings" is implied.